### ISSUE TITLE
Missing a language variable?

### ISSUE THREAD

**Issue by wads24** [andreaskou/phpbb_giphy#1]
PHPBB_SEARCH_GIPHY and SEARCHPHPBB_PAGE missing in language file
![Screenshot_20210327-122823_Chrome](https://user-images.githubusercontent.com/26773817/112732623-407e8880-8ef8-11eb-82fe-918bce925d8a.jpg)

**Comment by andreaskou** [comment]
Hi @wads24, this ext is incomplete. I do not have really time to spend on it.
Also pleas note that it uses my personal developer code for querying giphy. So if you are going to use it please create your own dev code on https://developers.giphy.com/

**Comment by wads24** [comment]
Yes, I did that part but language file doesn't exist.  Too bad you quit on it.  I was digging it...

I found another one though, not as good as yours though :)

**Comment by andreaskou** [comment]
Lol, my ext is terrible. What's good with it?
